### ISSUE TITLE
Request: Signed dumps of O1 Archive Nodes

### ISSUE THREAD

**Issue by yourbuddyconner** [MinaProtocol/mina#8326]
In order to fully sync an archive node, it is necessary to retrieve blocks that were produced `> 290 blocks ago` off-chain. Staking providers who are running their own infra would love a way to recover from a catastrophic failure of their Archive infrastructure using a trusted source of data. 

Additionally, this would help anyone wanting a full archive database going back to genesis who wasn't there in the first 290 blocks. 

Some things I would look for in an implementation: 
- A chain of sql dumps that can be imported sequentially
- Dumps are signed with an O1 Labs pubic key
- Dumps would be available in a publicly-accessible object storage bucket

Extra credit: 
- The integrity of the data is _verifiable_ such that these could be used as a source of truth to compare other archive backups to (since we don't have the blockchain to compare to).

**Comment by mrmr1993** [comment]
It might make sense to use the 'precomputed blocks' format for this, especially since we're already capturing it. 

Since these 'precomputed blocks' are encoding of the actual blocks, any individual block can be verified in exactly the same way as a block received over the network is -- by verifying its proof against its contents -- and without any privileged/trusted signer. These can be imported into an archive node either using a running daemon (`advanced archive-blocks`), or using the [`archive_blocks`](https://github.com/MinaProtocol/mina/blob/develop/src/app/archive_blocks/archive_blocks.ml) executable to place them in the db directly (this tool is packaged with the archive node IIRC).

I'll add an `advanced` command to do the verification, so that we support this fully as an option.

**Comment by yourbuddyconner** [comment]
Yeah, the precomputed blocks would work great for this use-case, especially if they can be verified individually! Thanks @mrmr1993

**Comment by shimkiv** [comment]
So the `archive-blocks` advanced command was added.
@mrmr1993 can you please check if we need to do something else here?

**Comment by dkijania** [comment]
Done. Signing is not needed since one can run replayer and verify data by themselves, i think.
